getting_started_tutorials/cudf_pandas_stocks_demo.ipynb

```diff
@@ -0,0 +1,7061 @@
+{
```

Comment:
<ul><li>need to update this link to <a href="https://storage.googleapis.com/rapidsai/colab-data/usa_stocks_30m.parquet" target="_blank">https://storage.googleapis.com/rapidsai/colab-data/usa_stocks_30m.parquet</a></li></ul><p> </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/rapidsai-community/showcase/pull/22/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='5'/>